ci/do_ci.sh

```diff
@@ -598,8 +599,10 @@ case $CI_TARGET in
               --//distribution:envoy-contrib-binary=//distribution:custom/envoy-contrib \
               //distribution:packages.tar.gz
         if [[ "${ENVOY_BUILD_ARCH}" == "x86_64" ]]; then
+            echo "Packages created: ${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
             cp -a bazel-bin/distribution/packages.tar.gz "${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
         else
+            echo "Packages created: ${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
```

Comment:
remove - same above

Reply:
probably rename to `packages.arm64.tar.gz`

Reply:
i think i prefer it the existing way without the debugging info